Tasks/PackerBuildV0/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -39,8 +43,8 @@
   "loc.input.help.additionalBuilderParameters": "In auto generated Packer template mode the task creates a Packer template with an Azure builder. This builder is used to generate a machine image. You can add keys to the Azure builder to customize the generated Packer template. For example setting ssh_tty=true in case you are using a CentOS base image and you need to have a tty to run sudo.<br/>To view/edit the additional parameters in a grid, click on “…” next to text box.",
   "loc.input.label.skipTempFileCleanupDuringVMDeprovision": "Skip temporary file cleanup during deprovision",
   "loc.input.help.skipTempFileCleanupDuringVMDeprovision": "During deprovisioning of VM, skip clean-up of temporary files uploaded to VM. Refer [here](https://www.packer.io/docs/builders/azure.html#skip_clean)",
-  "loc.input.label.imageUri": "Image URL",
-  "loc.input.help.imageUri": "Provide a name for the output variable which will store generated machine image URL.",
+  "loc.input.label.imageUri": "Image URL or Name",
+  "loc.input.help.imageUri": "Provide a name for the output variable which will store generated machine image url for unmanaged VM disk or image name for managed VM disk.",
```

Comment:
think about re-wording to "Provide a name for the output variable which will store generated machine image VHD url for un-managed VM image or image name for managed VM image."

Reply:
done